### ISSUE TITLE
"Expecting one more argument to ‘Boolean’" using `Boolean` in client code

### ISSUE THREAD

**Issue by th3coop** [morpheusgraphql/morpheus-graphql#583]
I get the same error as #277 but that's closed fixed so reporting again.

Morpheus version: 0.17.0

Error:
```
• Expecting one more argument to ‘Boolean’
  Expected a type, but ‘Boolean’ has kind ‘Bool -> ScalarValue’
• In the first argument of ‘Maybe’, namely ‘Boolean’
  In the type ‘(Maybe Boolean)’
  In the definition of data constructor ‘StarWarsFilmsArgs’
```

Schema, a modified Starwars.graphql (altered name to be able to attach here):
[starwars.txt](https://github.com/morpheusgraphql/morpheus-graphql/files/6133702/starwars.txt)

I changes line 3322 to `skip: Boolean`

Client code StarWars.hs:
[StarWars.txt](https://github.com/morpheusgraphql/morpheus-graphql/files/6133711/StarWars.txt)

To even get it to that point I had to add the following lines to the file:
`{-# LANGUAGE DataKinds #-}` at line 8
and
`import Data.Morpheus.Types (ScalarValue (..))` at line 23

Thanks in advance for any help!

**Comment by nalchevanidze** [comment]
Hi @th3coop . thanks. i will check it out.

**Comment by th3coop** [comment]
@nalchevanidze thanks for the follow up.  Will give that a try.

**Comment by nalchevanidze** [comment]
is fixed in #589. I also added a test to avoid this in the next versions.
